modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -351,42 +382,44 @@ const OrderableTable = ({
 				</ManagementToolbar.Container>
 
 				{items.length ? (
-					<ClayTable className="orderable-table">
-						<ClayTable.Head>
-							<ClayTable.Row>
-								<ClayTable.Cell className="drag-handle-cell" />
-
-								{fields.map((field) => (
-									<ClayTable.Cell
-										headingCell
-										key={field.name}
-									>
-										{field.label}
-									</ClayTable.Cell>
-								))}
-
-								{actions && (
-									<ClayTable.Cell className="actions-cell" />
-								)}
-							</ClayTable.Row>
-						</ClayTable.Head>
-
-						<ClayTable.Body>
-							<DndProvider backend={HTML5Backend}>
-								{items.map((item, index) => (
-									<OrderableTableRow
-										actions={actions}
-										fields={fields}
-										index={index}
-										item={item}
-										key={item.id || index}
-										onOrderChange={handleOnOrderChange}
-										query={query}
-									/>
-								))}
-							</DndProvider>
-						</ClayTable.Body>
-					</ClayTable>
+					<DndProvider backend={HTML5Backend}>
+						<Table
+							actions={actions}
+							fields={fields}
+							items={items}
+							onDragCrossover={({
+								draggedIndex,
+								targetIndex,
+							}: {
+								draggedIndex: number;
+								targetIndex: number;
+							}) => {
+								const orderedItems = items.slice(0);
```

Comment:
nit: if our aim is to just clone the items array, should we spread instead? `[...items]`? I know it's practically the same, but I generally see people spread items for cloning instead of a slice.

Also, what about renaming to `newOrdererItems`? Makes it a bit more clear that we are creating a new collection to provide to `setItems`

Reply:
I like the spread idea! Seems easier to understand at a glance.

I left in the `ordered` prefix, as `newOrdered` seems redundant to me. `new` could maybe make sense instead of `ordered`, but I slightly prefer `ordered` as it is more descriptive.